SPECS/python-pytorch/python-pytorch.spec

```diff
@@ -0,0 +1,112 @@
+%global _empty_manifest_terminate_build 0
+Summary:        Tensors and Dynamic neural networks in Python with strong GPU acceleration.
+Name:           pytorch
+Version:        1.13.1
+Release:        1%{?dist}
+License:        BSD-3
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Development/Languages/Python
+URL:            https://pytorch.org/
+Source0:        https://github.com/pytorch/pytorch/releases/download/v%{version}/pytorch-v%{version}.tar.gz#/pytorch-%{version}.tar.gz
+Source1:        %{name}-%{version}-submodules.tar.gz
+BuildRequires:  gcc
+BuildRequires:  gcc-c++
+Requires:       python3-numpy
+
+%description
+PyTorch is a Python package that provides two high-level features:
+- Tensor computation (like NumPy) with strong GPU acceleration
+- Deep neural networks built on a tape-based autograd system
+You can reuse your favorite Python packages such as NumPy, SciPy and Cython to extend PyTorch when needed.
+
+%package -n     python3-pytorch
```

Comment:
We don't have automatic generation of runtime requirements for python specs like a lot of other distros do. There seems to be some runtime requirements missing, judging by [upstream's `setup.py`](https://github.com/pytorch/pytorch/blob/master/setup.py#L1015-L1020) We should add those in manually.

Reply:
We do have `filelock` and `typing_extensions` packages so I added those two as **Requires**, but we don't have `networkx` and `sympy` in Mariner. Interestingly, the cpu-only version of the [upstream pytorch image](https://hub.docker.com/r/bitnami/pytorch) that we are building our pytorch cpu-only image after doesn't seem to have `networkx` or `sympy` included. The image with `cuda` however, has `networkx` so we can definitely include it in next iteration when we build pytorch with cuda.